pytest_translations/po_files.py

```diff
@@ -6,26 +6,26 @@
 
 
 class PoFile(File):
-    def __init__(self, path, parent):
-        super(PoFile, self).__init__(path, parent)
+    def __init__(self, fspath, parent):
+        super(PoFile, self).__init__(fspath, parent)
```

Comment:
if we're already simplifying `super` calls, let's do the same here:
```suggestion
        super().__init__(fspath, parent)
```